Sources/Plasma/PubUtilLib/plPipeline/GL/plShaderNode.cpp

```diff
@@ -86,6 +86,16 @@ ST::string plShaderContext::RenderNode(std::shared_ptr<plShaderNode> node, std::
             return var->name;
         }
         break;
+            
+        case kOutputVariable:
+        {
+            std::shared_ptr<plOutputVariableNode> var = static_pointer_cast<plOutputVariableNode>(node);
+
+            this->outputVariables.insert(var);
+
+            return var->name;
+        }
+        break;
```

Comment:
I feel bad about kOutput vs kOutputVariable. :( For explicit outputs there might be a better naming scheme.